### ISSUE TITLE
Method setEnabledOptions does not exist

### ISSUE THREAD

**Issue by jeijei4** [sa-si-dev/virtual-select#160]
Example:
```js
var enabledOptions = [2, 6, 9];

document.querySelector('#sample-select').setEnabledOptions(enabledOptions);

/** to enable all options */
document.querySelector('#sample-select').setEnabledOptions(true);
`

**Comment by sa-si-dev** [comment]
@jeijei4 I will implement it in the future release.

**Comment by sa-si-dev** [comment]
@jeijei4 This feature available in the release [v1.0.32](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.32)
